### ISSUE TITLE
premake-vcpkg downloads vcpkg.exe on Linux

### ISSUE THREAD

**Issue by arctii** [Gaztin/premake-vcpkg#1]
When running in a Linux environment, premake-vcpkg downloads a windows executable of vcpkg. Is this intended behaviour or is the module installed or used incorrectly? Mind you that I only started to dabble with premake a few hours ago as of writing this after furiously giving up all hope over CMake.

**Comment by JeppeSRC** [comment]
> When running in a Linux environment, premake-vcpkg downloads a windows executable of vcpkg. Is this intended behaviour....

Yes
```lua
local url = "https://github.com/microsoft/vcpkg-tool/releases/download/"..version_date.."/vcpkg.exe"
```

**Comment by arctii** [comment]
If it is possible, how would I make it download/ install/ use Linux a compatible vcpkg?
Do I (or can I) simply change the download URL?

**Comment by Gaztin** [comment]
After minimal looking, it doesn't seem like the vcpkg binary is included in the GitHub releases. Changing the URL won't work unfortunately. You can try using vcpkg standalone without this plug-in and link to them via premake normally until this has been fixed.

**Comment by arctii** [comment]
I came to the same conclusion and solution. In `premake-system.lua` I simply added the paths to the locally installed vcpkg's include and lib folders using `includedirs` and `libdirs`; this means I have direct access to all my vcpkg-installed libraries. An extension to this solution would be to install vcpkg directly under the .premake folder even. I have no idea if this solution breaks any premake conventions but it works so I will not complain.
